relayer/src/config.ts

```diff
@@ -1,97 +1,100 @@
+        chain: "Statemine",
+      },
+      {
+        url: "https://karura.api.onfinality.io/public",
+        paraId: 2000,
+        chain: "Karura",
+      },
+      {
+        url: "https://bifrost-parachain.api.onfinality.io/public",
+        paraId: 2001,
+        chain: "Bifrost",
+      },
+      {
+        url: "https://khala.api.onfinality.io/public",
+        paraId: 2004,
+        chain: "Khala Network",
+      },
       {
-        url: "wss://kusama-rpc.polkadot.io",
-        parachains: [
-          {
-            url: "https://kusama-statemine-rpc.paritytech.net",
-            paraId: 1000,
-            chain: "Statemine",
-          },
-          {
-            url: "https://karura.api.onfinality.io/public",
-            paraId: 2000,
-            chain: "Karura",
-          },
-          {
-            url: "https://bifrost-parachain.api.onfinality.io/public",
-            paraId: 2001,
-            chain: "Bifrost",
-          },
-          {
-            url: "https://khala.api.onfinality.io/public",
-            paraId: 2004,
-            chain: "Khala Network",
-          },
-          {
-            url: "https://shiden.api.onfinality.io/public",
-            paraId: 2007,
-            chain: "Shiden",
-          },
-          {
-            url: "https://moonriver.api.onfinality.io/public",
-            paraId: 2023,
-            chain: "Moonriver",
-          },
-          {
-            url: "https://calamari.api.onfinality.io/public",
-            paraId: 2084,
-            chain: "Calamari",
-          },
-          {
-            url: "https://spiritnet.api.onfinality.io/public",
-            paraId: 2086,
-            chain: "Kilt Spiritnet",
-          },
-          {
-            url: "https://basilisk.api.onfinality.io/public",
-            paraId: 2090,
-            chain: "Basilisk",
-          },
-          {
-            url: "https://altair.api.onfinality.io/public",
-            paraId: 2088,
-            chain: "Altair",
-          },
-        ]
+        url: "https://shiden.api.onfinality.io/public",
+        paraId: 2007,
+        chain: "Shiden",
       },
-    ],
-  };
-};
+      {
+        url: "https://moonriver.api.onfinality.io/public",
+        paraId: 2023,
+        chain: "Moonriver",
+      },
+      {
+        url: "https://calamari.api.onfinality.io/public",
+        paraId: 2084,
+        chain: "Calamari",
+      },
+      {
+        url: "https://spiritnet.api.onfinality.io/public",
+        paraId: 2086,
+        chain: "Kilt Spiritnet",
+      },
+      {
+        url: "https://basilisk.api.onfinality.io/public",
+        paraId: 2090,
+        chain: "Basilisk",
+      },
+      {
+        url: "https://altair.api.onfinality.io/public",
+        paraId: 2088,
+        chain: "Altair",
+      },
+    ]
+  },
+];
+
+const config = new Config({
+  accountSeed: process.env.ACCOUNT_SEED,
+  targetChainUrl: process.env.TARGET_CHAIN_URL,
+  sourceChains,
+});
+
+export default config;
```

Comment:
What about exporting the class instead and instantiating it where you need to use it instead of having single global instance with environment variables only?

Then `Config` class and `sourceChains` would be re-exported and I don't really see the point of having `IConfig` here.

It will help when you get to writing tests.

Reply:
do you mean smth like this?
```
import Config, { sourceChains } from "./config";

const config = new Config({
  accountSeed: process.env.ACCOUNT_SEED,
  targetChainUrl: process.env.TARGET_CHAIN_URL,
  sourceChains,
});
```

Reply:
Exactly